scripts/release

```diff
@@ -5,18 +5,18 @@ set -e
 RELEASED_LOG="/tmp/node-pending-changes.md"
 THIS_VERSION=$(./scripts/bump --this-version)
 
-# Finally need to update the full changelog
-./scripts/changelog
-git add CHANGELOG.md
-git commit -m "Update Changelog for Release $THIS_VERSION"
-git push origin master
-
 # publish to npm
 npm publish
 
 # create release
 hub release create -c -F $RELEASED_LOG $THIS_VERSION
 
+# Finally need to update the full changelog
```

Comment:
Unsure how this block got moved up, but the changelog generator needs the tag to exist in order to include the latest changes.